### DISCUSSION TITLE
Welcome to MarshalX Discussions!

### DISCUSSION THREAD

**Discussion by MarshalX** [MarshalX/MarshalX#6] [А вот другая!]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MarshalX** [comment]
в е л к о м е

**Reply by iamnalinor** [reply]
з д р а с т е

**Reply by MarshalX** [reply]
пол

**Reply by iamnalinor** [reply]
лоп
